_posts/2024-05-01-board-updates.md

```diff
@@ -0,0 +1,26 @@
+---
+author_avatar: defna
+author_name: Tim Schilling
+author_url: /about
+categories: ""
+date: 2024-05-01 06:00:00
+feature_image: 2017-07-20
+layout: post
+read_time: ""
+show_avatar: true
+show_related_posts: false
+title: "Board Member Update for May 2024"
+permalink: /announcements/2023/05/01/board-updates/
+---
+
+We have a new board member as of May 2024!
+
+## Outgoing Director
+
+We would like to take a moment to thank our outgoing director, Jennifer Myers for her contributions to our board during her years of service. Jen joined the board in 2021 and has been an invaluable asset to the board during her tenure as treasurer. <Need something here>. We are deeply grateful for the time and effort she has invested.
```

Comment:
```suggestion
We want to take a moment to thank our outgoing director and Treasurer, Jennifer Myers, for her contributions to our board during her years of service. Jen joined the board in 2021 and has been an invaluable asset to the board during her tenure. <Need something here>. We are deeply grateful for the time and effort she has invested.
```

"<Need something here>" - did you have thoughts on direction?

Reply:
I saw Josue's thank-you, "Josue joined DEFNA in 2018, was a DjangoCon US volunteer, and was no stranger to speaking at DjangoCon US. Josue has always been a voice for pushing DEFNA forward and has hosted [our YouTube interviews](https://www.youtube.com/watch?v=KMO4RR5ISRE&list=PL2NFhrDSOxgUwF244g835zO3ciMkKILyJ) for many years." and felt like we should match that tone here.

Reply:
This isn't flowing, but maybe it helps with filling in some gaps: 

While Jen never attended DjangoCon US, she helped us balance our books, handle finances, processing grants during our first online and our return to in-person events.

Reply:
I think we should go with Peter's.